drivers/clock_control/Kconfig.nrf

```diff
@@ -46,7 +46,7 @@ config CLOCK_CONTROL_NRF_K32SRC_EXT_FULL_SWING
 endchoice
 
 config CLOCK_CONTROL_NRF_K32SRC_RC_CALIBRATION
-	bool
+	bool "Enable LF clock calibration"
```

Comment:
if this is supposed to only be used in test (be hidden otherwise), this needs to be reflected in the prompt

Reply:
I have use cases where this could be the appropriate value for the application given the current capabilities.

Reply:
there is a use case: app core on nrf53.

Reply:
can you elaborate?

Reply:
on nrf53 only net has `TEMP` peripheral so it is should perform calibration and if board has no LFXO then app will have LFRC source but should disable calibration. We don't hit it now because on first revision calibration is not working and there are no boards without LFXO.

Reply:
For mine: I don't care about absolute timing accuracy because I'm not using bluetooth and the LFRC stability is good enough/compensated for through another means.  (I'd want the ability to turn calibration on and off because I might use Bluetooth perhaps once an hour, or only when a button is pressed, or...).

In general we should assume that if we need something tunable, somebody else might want it too.

Reply:
@nordic-krch looking at your response, it does not feel that this Kconfig has to be a generally-visible option to the app developer, as this appears to be strictly related to HW capabilities.

You can, instead, add dependencies to the symbol, reflecting the statements you made above, and add the prompt conditionally if you are building a test. (the latter is a de-facto acceptable option)

Reply:
what about @pabigot statement?
> I have use cases where this could be the appropriate value for the application given the current capabilities.

Could there be an application which does not want calibration? to safe memory?

Reply:
Let's try again; @ioannisg why do you *not* want developers to be able to control this?

Or do you just want it to be better documented what happens if you don't enable it?

Reply:
@ioannisg can you take a look? I've added help and default n for nrf53 app.